Kickstarter-iOS/Views/CommentCellHeaderStackView.swift

```diff
@@ -0,0 +1,186 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class CommentCellHeaderStackView: UIStackView {
+  // MARK: - Properties
+
+  fileprivate let viewModel = CommentCellViewModel()
+
+  private lazy var avatarImageView = { UIImageView(frame: .zero)
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var userNameLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var userNameTagLabel: PaddingLabel = { PaddingLabel(frame: .zero) }()
+  private lazy var postTimeLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var usernameLabelsStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var usernameTimeLabelsStackView: UIStackView = { UIStackView(frame: .zero) }()
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    self.bindStyles()
+    self.configureViews()
+    self.bindViewModel()
+  }
+
+  required init(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+    _ = self
+      |> \.axis .~ .horizontal
+      |> \.spacing .~ Styles.grid(2)
+
+    _ = self.usernameTimeLabelsStackView
+      |> \.axis .~ .vertical
+      |> \.spacing .~ Styles.grid(1)
+
+    _ = self.usernameLabelsStackView
+      |> \.axis .~ .horizontal
+      |> \.spacing .~ Styles.grid(1)
+      |> \.alignment .~ .leading
+
+    _ = self.userNameLabel
+      |> \.numberOfLines .~ 1
+      |> \.textColor .~ .ksr_support_700
+      |> \.textAlignment .~ .left
+      |> \.font .~ UIFont.ksr_callout().weighted(.semibold)
+      |> \.adjustsFontForContentSizeCategory .~ true
+
+    _ = self.userNameTagLabel
+      |> \.numberOfLines .~ 1
+      |> \.textColor .~ .ksr_create_500
+      |> \.textAlignment .~ .left
+      |> \.font .~ UIFont.ksr_callout().bolded
+      |> \.adjustsFontForContentSizeCategory .~ true
+
+    _ = self.postTimeLabel
+      |> \.numberOfLines .~ 1
+      |> \.textColor .~ .ksr_support_400
+      |> \.textAlignment .~ .left
+      |> \.font .~ UIFont.ksr_footnote()
+      |> \.adjustsFontForContentSizeCategory .~ true
+  }
+
+  // MARK: - Configuration
+
+  internal func configureWith(comment: DemoComment) {
+    self.viewModel.inputs.configureWith(comment: comment)
+  }
+
+  internal func configureUserTagStlye(from userTag: DemoComment.UserTagEnum) {
```

Comment:
Typo on the function name.

Also, usually we would output something like this from the VM and write tests for it. I guess if we are confident this is covered by snapshot tests we could forgo it here 🤔 @singhhari ?

Reply:
Yeah, I thought About this too, but I was thinking how to test this in the `vm`, and since snapshot test should cover it, I don't need it in the vm. 
But then we need to be confident that snapshot tests will cover it.

Reply:
I have moved this to the VM and provided tests for it.